src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -487,6 +487,99 @@ describe('StripeV3PaymentStrategy', () => {
+            const shippingAddressWithoutPhone = {
+                firstName: 'Test',
+                lastName: 'Tester',
+                company: 'Bigcommerce',
+                address1: '12345 Testing Way',
+                address2: '',
+                city: 'Some City',
+                stateOrProvince: 'California',
+                stateOrProvinceCode: 'CA',
+                country: 'United States',
+                countryCode: 'US',
+                postalCode: '95555',
+                shouldSaveAddress: true,
+                customFields: [],
+            };
+            const billingAddressWithoutPhone = {
+                id: '55c96cda6f04c',
+                firstName: 'Test',
+                lastName: 'Tester',
+                email: 'test@bigcommerce.com',
+                company: 'Bigcommerce',
+                address1: '12345 Testing Way',
+                address2: '',
+                city: 'Some City',
+                stateOrProvince: 'California',
+                stateOrProvinceCode: 'CA',
+                country: 'United States',
+                countryCode: 'US',
+                postalCode: '95555',
+                shouldSaveAddress: true,
+                customFields: [],
+            };
+
+            const customer = {
+                addresses: [{
+                    address1: 'TEST 123',
+                    address2: '',
+                    city: 'Netherlands',
+                    company: '',
+                    country: 'Netherlands',
+                    countryCode: 'NL',
+                    customFields: [],
+                    firstName: 'Test',
+                    id: 2,
+                    lastName: 'Test',
+                    postalCode: 'ASDF',
+                    stateOrProvince: '',
+                    stateOrProvinceCode: '',
+                    type: 'residential',
+                }],
+                email: 'test@bigcommerce.com',
+                firstName: 'Test',
+                fullName: 'Test Tester',
+                id: 1,
+                isGuest: false,
+                lastName: 'Tester',
+                shouldEncourageSignIn: false,
+                storeCredit: 0,
+            };
+
+            stripeV3JsMock.confirmCardPayment = jest.fn(
+                () => Promise.resolve(getConfirmPaymentResponse())
+            );
+
+            jest.spyOn(stripeScriptLoader, 'load').mockReturnValue(Promise.resolve(stripeV3JsMock));
+            jest.spyOn(stripeV3JsMock, 'elements').mockReturnValue(elements);
+            jest.spyOn(stripeV3JsMock.elements(), 'create').mockReturnValue(cardElement);
+            jest.spyOn(store.getState().customer, 'getCustomer').mockReturnValue(customer);
+            jest.spyOn(store.getState().shippingAddress, 'getShippingAddress').mockReturnValue(shippingAddressWithoutPhone);
+            jest.spyOn(store.getState().billingAddress, 'getBillingAddress').mockReturnValue(billingAddressWithoutPhone);
```

Comment:
I think there's no need to mock the shipping address as you are only testing the billing address.
```suggestion
            jest.spyOn(store.getState().billingAddress, 'getBillingAddress').mockReturnValue({ ...getBillingAddress(), phone: '' });
```